modules/pel/peltool/peltool.py

```diff
@@ -228,6 +228,9 @@ def parsePEL(stream: DataStream, config: Config, exit_on_error: bool):
     if config.non_serviceable_only and uh.isServiceable():
         return "", ""
 
+    if config.critSysTerm and uh.eventSeverity != 0x51:
```

Comment:
Please use constant with meaningful name for `0x51`

and have API like LN228 to check whether the PEL is critical or not to avoid changing condition in many places in the future if required.